mqemitter-mongodb.js

```diff
@@ -149,6 +150,37 @@ function MQEmitterMongoDB (opts) {
 
 inherits(MQEmitterMongoDB, MQEmitter)
 
+MQEmitterMongoDB.prototype._findNext = function(id) {
+  for (var i = 0, len = this._queue.length; i < len; i++) {
+    if(this._queue[i]._id.toString() === id) {
```

Comment:
these are a lot of .toString() operations for ._id. Maybe we can store it in already in string form?

Reply:
I know but it must be an ObjectId when I assign it to lastId to make the checks in emit, the alternative would be to store it in string format and convert it to objectId when I assign it to lastId but I don't think there is any optimization in this as there would be lot `new ObjectId(id)` in place of `.toString()`. An alternative would be to add another property like `_idString` and use that to compare

Reply:
Also, in 99.99% of cases the _findNext will always return 0 so it will do only one check, the 0.01% will return 1

Reply:
Give that we store it in an object, we can just store it in both formats.

Note that we use it here and then in `_emitPacket` as well.